roles/edpm_reboot/tasks/main.yaml

```diff
@@ -53,10 +53,15 @@
     paths: /var/lib/openstack/reboot_required/
   register: reboot_files
 
-- name: Check if node has a nova.conf config
+- name: Check if old_tripleo_node config dir exists
   ansible.builtin.stat:
-    path: "{{ edpm_reboot_nova_compute_config_dir }}/01-nova.conf"
-  register: nova_conf_check
+    path: "{{ edpm_reboot_old_tripleo_node_config_dir }}"
+  register: old_tripleo_node_conf
```

Comment:
ok so this is trying to skip reboot if there is tripleo content.

we don't every delete those config files during adoption so this will mean we will never reboot. compute nodes in auto mode for any adopted nodes.

Reply:
if there is a jira to plan a post adoption task to clean up these stale config i think we can also ignore this point and proceed with the pr

Reply:
Auto mode is there only for greenfield deployment only, so regardless if cleanup happen(it should happen I agree) adopted nodes won't be rebooted in auto mode anyway(after adoption they will have new configs generated).

Reply:
So that is problematic because after a deployment is adopted it is ment to be identical to a greenfield deployment with regards to future day 2 operation. So post adoption completing auto should work the same for greenfield and adopted node otherwise we have a big.

Reply:
that is correct, it seems that users then would need to change all osdpns ansible vars for that policy to auto and / or use this value for the future osdp deployments, on "day 2", post-adoption. That is a docs impact though, nothing for this code to change